frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts

```diff
@@ -0,0 +1,670 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```

Comment:
Please make sure we are disabling eslint only in specific cases

Reply:
Done